src/Orchard.Cms.Web/Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -10,9 +10,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link type="image/x-icon" rel="shortcut icon" href="/TheTheme/Content/favicon.ico" />
 
-    <script asp-name="bootstrap" use-cdn="true" at="Foot"></script>
+    <script asp-name="bootstrap" use-cdn="true" version="3.3.7" at="Foot"></script>
```

Comment:
I think you should only specify 3.3-* or maybe 3-* if it works. This way you don't change the theme when a new bs minor version is shipped.